### PR TITLE
Add a half-page ad

### PR DESCRIPTION
**Pull request by charypar**
Depends on Financial-Times/next-ads-component#69. That needs to be reviewed and shipped just before this is.

### CODE REVIEW

File: views/front-page.html [hunk 1]
```diff
@@ -12,11 +12,11 @@
 	<a id="fastft-tab" class="nh-header-tabs__marker" >fastFT</a>
 	<div class="o-grid-container fixed-width-container">
 		<div class="o-grid-row">
-			<div class="main-content nh-header-tabs__panel" data-o-grid-colspan="12 L9">
+			<div class="main-content nh-header-tabs__panel" data-o-grid-colspan="12 L8 XL9">
 				<h1 class="n-util-visually-hidden" id="lead-pod-title">Top stories</h1>
 				{{> lead-today leads=content.top.leads liveBlogs=content.top.liveBlogs items=content.top.items popularTopics=content.popularTopics }}
 			</div>
-			<aside class="sidebar sidebar--fastft nh-header-tabs__panel" data-trackable="fastft" data-o-grid-colspan="12 L3" role="presentation">
+			<aside class="sidebar sidebar--fastft nh-header-tabs__panel" data-trackable="fastft" data-o-grid-colspan="12 L4 XL3" role="presentation">
```
Comment (ironsidevsquincy) [inline views/front-page.html]: to keep it consistent with the other ads, might be worth putting a specific div in here that you want the ad to go into

`<div class="ad-sidebar-container"></div>`, and update the ads-component selector

Comment (charypar) [inline views/front-page.html]: Yeah, will do

**Comment by ironsidevsquincy** [comment]
:+1:

**Comment by charypar** [comment]
Need to add a feature flag for this

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**